Source/FieldSolver/SpectralSolver/SpectralSolver.H

```diff
@@ -83,7 +83,10 @@ class SpectralSolver
              algorithm->CurrentCorrection( field_data, current, rho );
         };
 
+        bool fft_do_time_averaging = false;
```

Comment:
Here I would prefix this variable with `m_`, as it is a member of a class, following WarpX's style guidelines:
```suggestion
        bool m_fft_do_time_averaging = false;
```

Reply:
@EZoni By "here" you mean only in this class? Because then I would change all such occurrences, e.g. I have another one in WarpX class. In general, I didn't see this guidance followed strictly in WarpX, hence decided to shorten the name.